src/schemas/map.json

```diff
@@ -28,19 +28,89 @@
     "ItemsPath": {
       "$ref": "paths.json#/definitions/asl_ref_path"
     },
+    "Label": {
+      "type": "string"
+    },
     "MaxConcurrency": {
       "type": "number",
       "minimum": 0
     },
+    "ItemReader": {
+      "$comment": "need to improve this type",
+      "type": "object"
+    },
```

Comment:
I believe the fields for this are defined in the spec and don't vary. Easy enough to update to define more here.